### PR TITLE
Spark Summit 2017 Sample Event

### PR DESCRIPTION
**Pull request by arp95**
Adding information for spark sample event in the next few days.

### CODE REVIEW

**Comment by arp95** [comment]
@iamareebjamal @mahikaw @Shailesh351 @shreyasnbhat
please keep on checking the files when you find time. Will keep on adding more info but if there is a mistake its better to point out now.

**Comment by mariobehling** [comment]
@arp95 Please take out WIP if this PR is ready for review. Please help to speed up the PR process and finalize PRs in a timely manner.

**Comment by arp95** [comment]
@mariobehling yeah sure. just a day or two more to complete the stuff.

**Comment by arp95** [comment]
@iamareebjamal @mahikaw @Shailesh351 @shreyasnbhat 
please review.

File: sample/SparkSummit_17/config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "Email": "spark@gmail.com",
+  "App_Name": "Spark Summit 2017",
+  "Api_Link": "https://raw.githubusercontent.com/arp95/open-event/spark/sample/spark"
```
Comment (iamareebjamal) [inline sample/SparkSummit_17/config.json]: The API link isn't correct. Please fix

Should be https://raw.githubusercontent.com/arp95/open-event/spark/sample/SparkSummit_17

File: sample/SparkSummit_17/event [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            "name": "Twitter"
+        }
+    ], 
+    "ticket_url": "https://prevalentdesignevents.com/sparksummit/ss17/", 
+    "privacy": "public", 
+    "type": "Conference", 
+    "topic": "Data Science", 
+    "sub_topic": "Other", 
+    "code_of_conduct": "", 
+    "copyright": "",
```
Comment (iamareebjamal) [inline sample/SparkSummit_17/event]: This should be object, not string

**Comment by arp95** [comment]
@iamareebjamal @mahikaw @Shailesh351 @shreyasnbhat 
finally the app is working! 
Please test as soon as you guys get time.

**Comment by iamareebjamal** [comment]
@arp95 Testing

**Comment by mahikaw** [comment]
@arp95 The sample looks great to me.
Although, I came across sessions that don't have speakers. Do you think you could add their speakers?
And yeah, some JSON files(sessions, micro locations etc.) need to be formatted. You can easily achieve that using any online tool. 
Nice work!  👍

**Comment by arp95** [comment]
mahika those sessions are training ones. they dont have speakers. else its fine?
if yes please approve.

**Comment by arp95** [comment]
@Shailesh351 @shreyasnbhat please review

**Comment by mahikaw** [comment]
@arp95 please do the formatting as well. You could use sublime or any other online tool.

**Comment by arp95** [comment]
@mahikaw please point out where the formatting is wrong?

**Comment by arp95** [comment]
@Shailesh351
yeah you are right. Sending the pr with the correction.

**Comment by arp95** [comment]
@mariobehling @niranjan94 please have a final look. i think its safe to merge.

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**